tripal_jbrowse_mgmt/includes/tripal_jbrowse_mgmt_tracks.form.inc

```diff
@@ -367,3 +367,133 @@ function tripal_jbrowse_mgmt_delete_track_form_submit($form, &$form_state) {
 
   drupal_goto('admin/tripal/extension/tripal_jbrowse/management/instances/' . $track->instance_id);
 }
+
+/**
+ * Track json editor advance form.
+ * allow user to make all configurations of a track
+ *
+ * @param array $form
+ * @param array $form_state
+ * @param int $track_id
+ *
+ * @return array
+ * @throws \Exception
+ */
+function tripal_jbrowse_mgmt_json_editor_advance_form($form, &$form_state, $track_id) {
+  $track = tripal_jbrowse_mgmt_get_track($track_id);
+
+  if (!$track->id) {
+    $form['error'] = [
+      '#type' => 'item',
+      '#markup' => '<p style="color: red">Unable to find track.</p>',
+    ];
+
+    return $form;
+  }
+
+  $instance = tripal_jbrowse_mgmt_get_instance($track->instance_id);
+  $json = tripal_jbrowse_mgmt_get_json($instance);
+  $form_state['track_json'] = $json;
+
+  $key = tripal_jbrowse_mgmt_make_slug($track->label);
+  $track_json = NULL;
+  $track_index = NULL;
+  foreach ($json['tracks'] as $index => $jtrack) {
+    if ($jtrack['label'] === $key) {
+      $track_json = $jtrack;
+      $track_index = $index;
+      break;
+    }
+  }
+
+  if (!$track_json) {
+    $form['error'] = [
+      '#type' => 'item',
+      '#markup' => '<p style="color: red">Unable to find track in json!</p>',
+    ];
+    return $form;
+  }
+
+  $form['track_index'] = [
+    '#type' => 'hidden',
+    '#value' => $track_index,
+  ];
+
+  $form['track_id'] = [
+    '#type' => 'hidden',
+    '#value' => $track->id,
+  ];
+
+  $instr_detail = '<strong>Only use this function if wanted configuration option is not included in Track Manage.</strong><br>Details of JBrowse track configuration can be found in '.l('JBrowse Documentation', 'http://jbrowse.org/docs/canvas_features.html').'.
+    <br><strong>Please</strong> be extra cautious while editing track configuration since it will make changes in file trackList.json directly.<br>';
```

Comment:
```suggestion
    <div class="messages warning">Please be extra cautious while editing track configuration, since it will make changes in file trackList.json directly. Also, do not change the "label" since it will disconnect this track from the JBrowse Management form.</div>';
```